package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "camelcase-keys",
-	"version": "9.0.0",
+	"version": "10.0.0",
```

Comment:
This should be done by the maintainer (me).

Reply:
Sorry 👼🏻. Reverted to 9.0.0.